app/routes/app+/market_.$address.submission.$submissionId.tsx

```diff
@@ -94,7 +99,20 @@ export default function ChallengeSubmission() {
   const isWinner = false;
 
   return (
-    <Container className="py-16 px-10">
+    <Container className="pt-9 pb-16 px-10">
+      <div className="flex gap-3.5 text-stone-500 pb-12 items-center">
```

Comment:
Should be pretty easy to componentize into a Breadcrumbs component. I would make a prop that is an array of objects like:

```js
[{ link: `/app/${laborMarket.appData?.type}`, name: "Marketplaces"}, /*...*/]
```